src/Core/Domain/Address/Command/AddManufacturerAddressCommand.php

```diff
@@ -234,6 +242,14 @@ public function getOther()
         return $this->other;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getDni()
```

Comment:
```suggestion
    public function getDni(): ?string
```

Reply:
same we can't type parameters for 1.7.6.x

Reply:
We can't in 176 (PHP5)